common/util/rviz_plugins/autoware_vehicle_rviz_plugin/src/tools/steering_angle.cpp

```diff
@@ -162,8 +159,7 @@ void SteeringAngleDisplay::processMessage(
 
   QMatrix rotation_matrix;
   rotation_matrix.rotate(
-    static_cast<qreal>(
-      std::round(property_handle_angle_scale_->getFloat() * (msg_ptr->data / M_PI) * -180.0)));
+    std::round(property_handle_angle_scale_->getFloat() * (msg_ptr->data / M_PI) * -180.0));
```

Comment:
[Question] Why `static_cast` was removed? A lint issue?

Reply:
I thought this was one cause of rviz crush problem, but I think this is also my misunderstanding.